transactionoutbox-core/src/test/java/com/gruelbox/transactionoutbox/TestDefaultPersistorConfiguration.java

```diff
@@ -35,6 +38,30 @@ final void whenMigrateIsFalseDoNotMigrate() throws Exception {
         });
   }
 
+  @Test
+  final void writeSchema() {
+    StringWriter stringWriter = new StringWriter();
+
+    DefaultPersistor defaultPersistor = DefaultPersistor.builder().dialect(Dialect.H2).build();
+    defaultPersistor.writeSchema(stringWriter);
+
+    String migrations = stringWriter.toString();
+
+    assertThat(migrations, startsWith("-- 1: Create outbox table"));
+    assertThat(
+        migrations,
+        containsString(
+            "-- 2: Add unique request id"
+                + System.lineSeparator()
+                + "ALTER TABLE TXNO_OUTBOX ADD COLUMN uniqueRequestId VARCHAR(100) NULL UNIQUE"));
+    assertThat(
+        migrations,
+        containsString(
+            "-- 8: Update length of invocation column on outbox for MySQL dialects only."
+                + System.lineSeparator()
+                + "-- Nothing for H2"));
```

Comment:
A few assertions.

Asserting everything might become annoying when new migrations are added. On the other hand, I guess that does not happen that often.

I can of course change this to be a full assertion. Maybe even do the full assertion for each dialect.